From 7bd01196f36cfec63d7002fb1de105621cadd5d2 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 7 Jul 2022 23:50:07 +0200
Subject: [PATCH 63/N] fixup! Temporarily decrement the DLL version

Let's undo this, for two reasons:

- since we rebased to v3.3.5, it was incorrect to decrement the DLL
  version, and

- we're about to pull in an intermediate change (again) which will claim
  that we're at v3.3.6, and we'll need to decrement _that_.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/cygwin/include/cygwin/version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/cygwin/include/cygwin/version.h b/winsup/cygwin/include/cygwin/version.h
index d783447..ba645e4 100644
--- a/winsup/cygwin/include/cygwin/version.h
+++ b/winsup/cygwin/include/cygwin/version.h
@@ -11,7 +11,7 @@ details. */
    changes to the DLL and is mainly informative in nature. */
 
 #define CYGWIN_VERSION_DLL_MAJOR 3003
-#define CYGWIN_VERSION_DLL_MINOR 4
+#define CYGWIN_VERSION_DLL_MINOR 5
 
 /* Major numbers before CYGWIN_VERSION_DLL_EPOCH are incompatible. */
 
-- 
2.9.0

